src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -4617,6 +4617,18 @@ SyntaxKind.FileScopedNamespaceDeclaration or
 
             AddInitializers(builder.InstanceInitializers, instanceInitializers);
             AddInitializers(builder.StaticInitializers, staticInitializers);
+
+            // local functions
```

Comment:
Nit: Comment isn't necessary (I think the IDE team normally does this, but we don't).